Makefile

```
@@ -80,6 +86,7 @@ uninstall:
 	@rm -f "$(DESTDIR)/etc/conf.d/grub-btrfsd;"
 	@rm -f "$(LIB_DIR)/initcpio/install/grub-btrfs-overlayfs"
 	@rm -f "$(LIB_DIR)/initcpio/hooks/grub-btrfs-overlayfs"
+	@initramfs/ubuntu/setup uninstall
```

This will make he Makefile fail and exit if it can't find `initramfs/ubuntu/setup`. (Which it wont on any distro that isn't Ubuntu or Ubuntu based.)

```suggestion
	@initramfs/ubuntu/setup uninstall || true
```

---

Thanks, I should have thought of that.
Resolved?

---

> Resolved?

Sure is. 
Make can be a life saver but also a real clusterfuck sometimes